infra/aws/cloudformation/iam/cdn.packages.k8s.io/cloudformation.yaml

```diff
@@ -0,0 +1,210 @@
+      Roles:
+        - Ref: ProvisionerRole
+    DependsOn: ProvisionerRole
+
+  # ProvisionerApplyAccessPolicy is an IAM policy that grants permissions to run `terraform apply`
+  ProvisionerApplyAccessPolicy:
+    Type: 'AWS::IAM::ManagedPolicy'
+    Properties:
+      ManagedPolicyName: ProvisionerApplyAccess
+      PolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Effect: 'Allow'
+            Action:
+              - 'acm:AddTagsToCertificate'
+              - 'acm:RequestCertificate'
+            Resource: !Sub 'arn:aws:acm:us-east-1:${AWS::AccountId}:certificate/*'
+          - Effect: 'Allow'
+            Action:
+              - 'cloudfront:CreateDistribution'
+              - 'cloudfront:CreateOriginAccessControl'
+              - 'cloudfront:UpdateDistribution'
+              - 'cloudfront:TagResource'
+            Resource:
+              - !Sub 'arn:aws:cloudfront::${AWS::AccountId}:distribution/*'
+              - !Sub 'arn:aws:cloudfront::${AWS::AccountId}:cache-policy/*'
+              - !Sub 'arn:aws:cloudfront::${AWS::AccountId}:origin-access-control/*'
+          - Effect: 'Allow'
+            Action:
+              - 's3:CreateBucket'
+              - 's3:PutBucketOwnershipControls'
+              - 's3:PutBucketPolicy'
+              - 's3:PutBucketTagging'
+            Resource: 'arn:aws:s3:::*-cdn-packages-k8s-io-*'
+          - Effect: 'Allow'
+            Action:
+              - 's3:PutObject'
+            Resource: !Sub 'arn:aws:s3:::${TFStateBucket}/*'
+      # Attach the policy to the provisioner IAM user
+      Roles:
+        - Ref: ProvisionerRole
+    DependsOn: ProvisionerRole
+
+  # ProvisionerDestroyAccessPolicy is an IAM policy that grants permissions to run `terraform destroy`
+  ProvisionerDestroyAccessPolicy:
+    Type: 'AWS::IAM::ManagedPolicy'
+    Properties:
+      ManagedPolicyName: ProvisionerDestroyAccess
+      PolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Effect: 'Allow'
+            Action:
+              - 'acm:DeleteCertificate'
+            Resource: !Sub 'arn:aws:acm:us-east-1:${AWS::AccountId}:certificate/*'
+          - Effect: 'Allow'
+            Action:
+              - 'cloudfront:DeleteDistribution'
+              - 'cloudfront:DeleteOriginAccessControl'
+            Resource:
+              - !Sub 'arn:aws:cloudfront::${AWS::AccountId}:distribution/*'
+              - !Sub 'arn:aws:cloudfront::${AWS::AccountId}:cache-policy/*'
+              - !Sub 'arn:aws:cloudfront::${AWS::AccountId}:origin-access-control/*'
+          - Effect: 'Allow'
+            Action:
+              - 's3:DeleteBucket'
+              - 's3:DeleteBucketPolicy'
+            Resource: 'arn:aws:s3:::*-cdn-packages-k8s-io-*'
+      # Attach the policy to the provisioner IAM user
+      Roles:
+        - Ref: ProvisionerRole
+    DependsOn: ProvisionerRole
+
+  #########################################
+  # IAM user for OBS                      #
```

Comment:
What's OBS? Let's make it clear we don't mean https://obsproject.com/

Reply:
Nice catch, I almost everywhere used `OpenBuildService (OBS)` to make a clear distinction.